maven-plugin/src/main/scala/akka/grpc/maven/AbstractGenerateMojo.scala

```diff
@@ -186,7 +189,7 @@ abstract class AbstractGenerateMojo @Inject() (buildContext: BuildContext) exten
 
       val runProtoc: Seq[String] => Int = args =>
         com.github.os72.protocjar.Protoc.runProtoc(protocVersion +: args.toArray)
-      val protocOptions = Seq.empty
+      val protocOptions = if (includeStdTypes) Seq("--include_std_types") else Seq.empty
```

Comment:
(another option would be to allow arbitrary additional protoc options, but this is :+1:)

Reply:
this will be a bit more tough as the java library expect the arguments as if they were received from the command line. 
I want to avoid this. Besides, this can allow user to specify some of the arguments twice